site/content/extend/plugins/webapp/proper-configuration.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Proper Configuration
+subsection: Web App Plugins
+weight: 0
+---
+
+### Developing in the plugin's webapp folder
+
+In order for your IDE to know the root directory of the plugin's webapp code, it is advantageous to open the IDE in the webapp folder itself when working on the webapp portion of the plugin. This way, the IDE is aware of files such as `webpack.config.js` and `tsconfig.json`.
```

Comment:
Following from above, in this case I would say this should be "webapp" as well, as it's referring to the naming convention of a repo?